### ISSUE TITLE
readData function not work

### ISSUE THREAD

**Issue by dharmesh8b** [vackosar/keras-punctuator#7]
Hi, I am using punctuator.py script for training. I am using the same example that you mention in script main function (news.2011.en.shuffled). But in sampleData function, there is the call of readData function for reading words from the text. In the same function take more time and memory after that give the error segment fault and core dump. And the programme is stopped running. so can you please advice what is the problem in this function? I am using, tensorflow 1.8.0, keras 2.1.6 , python 2.7.14.

**Comment by vackosar** [comment]
try reducing amount samples to process. I think there is a variable or parameter for that somewhere. If you can get successful run win say 1000 samples, then the issue is most likely not enough memory on your system for the sample count you had.
